apps/lend/src/components/AlertLoanSummary/components/SummaryFull.tsx

```diff
@@ -48,16 +53,28 @@ const SummaryFull = ({
       {title}
 
       <Item $minWidth={minWidth} label={t`Debt:`} value={`${format(stateDebt)} ${borrowedSymbol}`} />
+
       {+receive > 0 ? (
```

Comment:
Nested ternary operators are very difficult to read. It may be easier to do with ordinary binary conditions.
```suggestion
      {+receive > 0 && ( ... )}
      {+receive <= 0 && +stateBorrowed > 0 && ( ... )}
```